R/db.R

```diff
@@ -159,8 +159,8 @@ write_data_safely <- function(dbconn, tblname, df, key) {
     df[, id_cols] <- df[, id_cols_ix]
 
     # Convert some types to character so they go in the DB properly.
-    to_chars <- vapply(df, function(x) is.factor(x) || is.ordered(x) || is.logical(x), logical(1))
-    df[, to_chars] <- lapply(df[, to_chars], as.character)
+    to_chars <- unname(vapply(df, function(x) is.factor(x) || is.ordered(x) || is.logical(x), logical(1)))
+    df[, to_chars] <- sapply(df[, to_chars], as.character, USE.NAMES = FALSE)
```

Comment:
I'm confused. Why is this an sapply? `lapply` should work fine here and is usually the correct approach. Can you s3store an example `df` and `to_chars`?